### ISSUE TITLE
Extra tab inserted below comment

### ISSUE THREAD

**Issue by SolderedMushroom** [dotnet/roslyn#17585]
I realize the following might be a VS editor issue. However, I vaguely remember seeing this reported in the Roslyn repo a while ago, and I also vaguely remember it was supposed to have been fixed for VS2017. If I'm imagining all of that, and/or this is not the correct place for this bug, feel free to close the issue.

**Version Used**: 

Enterprise 2017 (15.0.0-RTW+26228.4)

**Steps to Reproduce**:

Type the following:

```csharp
public void Method()
{

}

// comment$$

public void AnotherMethod()
{

}
```

Place the cursor after the comment and press Enter twice.

**Actual Behavior**:

An extra tab is inserted upon entering the second newline.

![image](https://cloud.githubusercontent.com/assets/10489260/23667544/373c5ef2-032c-11e7-9329-d7ff21723101.png)

Note that the comment must be in between two methods, two namespaces, etc. This doesn't happen (for example) within a method body, or when the comment comes before any  methods in a class declaration.

**Comment by Pilchie** [comment]
Duplicate of https://github.com/dotnet/roslyn/issues/3321

**Comment by SolderedMushroom** [comment]
@Pilchie Thanks. I see it was filed two years ago, no wonder my recollection was so foggy.
